### PR TITLE
feat(core): adds intellij platform 2023.1 or later support

### PR DESCRIPTION
**Pull request by pavankjadda**

### CODE REVIEW

**Review by fmbetancourt — APPROVED** [review]

**Comment by darthShadow** [comment]
Hi @hsz, sorry for the ping, but would it possible for you to take a look at this PR and merge it if there are no issues?

**Comment by hsz** [comment]
Sorry for the long delay.
Unfortunately, it's impossible to release a plugin just with such a since/until build numbers bumped.
This is because several incompatibilities with the 2023.1 platform prevent the update from being published.

I'm currently working on the 2.0 release (unfortunately moving to a new house at the same time) — I hope to work in the following days to finally make it available for everyone.

**Comment by 13steinj** [comment]
@hsz any update? No rush; just the .ignore plugin still advertises this (and with this plugin now incompatible, just leads to confusion).

**Comment by darthShadow** [comment]
Hi @hsz, sorry for the ping again, but do you have any update for us on this? I would like to avoid switching to a manually-installed jar file.

**Comment by 13steinj** [comment]
main reason this is problematic is the .ignore plugin points to this one...which then can't be installed

I would have assumed there's a way to specify the version range in the config to just keep it installable

File: gradle.properties [hunk 1]
```diff
@@ -4,22 +4,22 @@ pluginGroup = ski.chrzanow.foldableprojectview
 pluginName = Foldable ProjectView
 pluginRepositoryUrl = https://github.com/hsz/intellij-foldable-projectview
 # SemVer format -> https://semver.org
-pluginVersion = 1.1.4
+pluginVersion = 1.1.5
 
 # Supported build number ranges and IntelliJ Platform versions -> https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
-pluginSinceBuild = 221
+pluginSinceBuild = 231
 pluginUntilBuild = 231.*
```
Comment (LecrisUT) [inline gradle.properties:11]: @13steinj see here. But @hsz said they have plans for a 2.0 version

**Comment by hvsy** [comment]
> Sorry for the long delay. Unfortunately, it's impossible to release a plugin just with such a since/until build numbers bumped. This is because several incompatibilities with the 2023.1 platform prevent the update from being published.
> 
> I'm currently working on the 2.0 release (unfortunately moving to a new house at the same time) — I hope to work in the following days to finally make it available for everyone.

Can't wait to use the 2.0 version!!!!!

**Comment by pavankjadda** [comment]
> @pavankjadda Your release does not seem to work for me
> 
> 
> 
> Screenshot 2023-07-25 at 11 07 28
> 
> 

For 2023.2 you need to use version 1.1.6. For 2023.1 you need to use 1.1.5

**Comment by ackmanx** [comment]
> 

Brilliant! You're a lifesaver 😃

**Pull request closed without merging**